OWNERS_ALIASES

```diff
@@ -0,0 +1,13 @@
+# Sort the member alphabetically.
+aliases:
+  sig-critical-approvers-config-components:
+    - easonn7
+    - kevin-xianliu
+    - zhangjinpeng87
+  sig-critical-approvers-config-src:
+    - easonn7
+    - kevin-xianliu
+    - cfzjywxk
```

Comment:
\+ zhangjinpeng87 
and 
+ [Connor1996](https://github.com/Connor1996)
+ [overvenus](https://github.com/overvenus)
I think